strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/cron/jobs/DownloadRemoteMavenIndexCronJobTestIT.java

```diff
@@ -78,6 +79,7 @@
     @Test
     @ExtendWith({ RepositoryManagementTestExecutionListener.class,
                   ArtifactManagementTestExecutionListener.class })
+    @Disabled
```

Comment:
Is there a follow-up task for sorting this test out? If no, could you please create one? Thanks!

Reply:
yes, I've already mentioned this

Reply:
Great! Thanks!